docs/_documentations/remote-deploying-codewind.md

```diff
@@ -164,15 +164,27 @@ Your Codewind instance is deployed. Access it by way of the **Access URL** retur
 
 **Note:** When you are installing on Windows, the script runs and then closes the Git Bash window. To retrieve the Access URLs for Codewind, enter the following command:
 
-`$ kubectl get codewind -n codewind`
+`$ kubectl get codewinds -n codewind`
 
 This command returns output in the format of the following example:
 
 ```
-NAME       USERNAME   NAMESPACE   AGE    KEYCLOAK   REGISTRATION   ACCESSURL
-devex001   jane       codewind    119m   devex001   Complete       https://codewind-gatekeeper-devex001.<ingress-domain>
+NAME    USERNAME   NAMESPACE   WORKSPACE      AGE    KEYCLOAK   REGISTRATION   ACCESSURL
+jane1   jane       codewind    kbc3b0x2qins   119m   devex001   Complete       https://codewind-gatekeeper-kbc3b0x2qins.codewind.<ingress-domain>
 ```
 
+For the 0.14.0 release of Codewind, a new `WORKSPACE` field appears in the output. This field is a randomly generated set of characters that are used in both the URL of the Gatekeeper and the name of the pods and resources that you create. 
+
+To help you to determine the unique resources for your deployment, you can use the `NAME` field or you can use the new workspace ID given in the `WORKSPACE` field. All of your Codewind deployments have both the `NAME` and `WORKSPACE` as labels for you to use. 
+
+Also, new for 0.14.0, the namespace appears in the `ACCESSURL`. The namespace label is to enable the Codewind Operator to deploy into different namespaces. In the example, the namespace is `codewind` and it appears just before `<ingress-domain>`. 
```

Comment:
We don't yet support the operator running in different namespaces.  Right now we insist the namespace must be "codewind". This change of adding the namespace is part of our journey towards allowing different namespaces but we're not finished yet.

Reply:
Changed the sentence to read "Also, new for 0.14.0, the `codewind` namespace appears in the `ACCESSURL`. In the example, it appears just before `<ingress-domain>`. "